language.md

```diff
@@ -351,6 +351,40 @@ deduplicate it from other alerts. The `Annotations` section define information
 that is passed to Alert Manager that gets overwritten. For details, see the
 [Alert Manager](https://prometheus.io/docs/alerting/clients/) documentation.
 
+## Testing
+Shesmu provides three ways to check a script:
+
+- uploading a script using the UI (best for users)
+- uploading a script to a remote server (best for automation, including presubmit checks)
+- using a local copy of the Shesmu JAR and a remote server (helpful if changing the Shesmu language)
+
+To use the UI, from the Shesmu server page, choose _Tools_ then _Olive
+Checker_. Write script in the box or use the upload the button and then hit
+check. If successful, the metro digrams from the olive dashboard will be
```

Comment:
"check" -> `Check` ? (makes it clearer that it's a button)
How about this for the last sentence: If the file is a valid Shesmu script, a diagram (like those from the olive dashboard) will be displayed for each olive.